tasks/create-advisory/create-advisory.yaml

```diff
@@ -65,7 +66,15 @@ spec:
         configMapName=$(jq -er '.sign.configMapName' "$(workspaces.data.path)/$(params.dataPath)")
 
         # Validate type
-        advisoryType=$(jq -r ".type" <<< "$advisoryData")
+        advisoryType=$(jq -r '.type' <<< "$advisoryData")
+        if [ "$advisoryType" == "null" ]; then
```

Comment:
Any reason why not to just use the jq default mechanism?

Reply:
using the jq default mechanism will only give me `advisoryType` with the default value.

I then need to patch `advisoryData` with the updated value later.

is there a better way?